Orange/widgets/data/owselectrows.py

```diff
@@ -427,8 +449,14 @@ def commit(self):
                             self.error(e.args[0])
                             return
 
+                    try:
+                        floats = [float(v) for v in values]
```

Comment:
What about putting all the parsing (time and ordinary) to a separate method that returns a list of float values or None if the parsing failed? This would improve the readability a bit and probably pass the failing travis test.